datayoga-py/datayoga/jmespath_custom_functions.py

```diff
@@ -86,7 +86,8 @@ def prepare() -> Union[bytes, bytearray]:
             if isinstance(obj, str):
                 return obj.encode()
 
-            return orjson.dumps(obj)
+            # the 'separators' arg is needed to remove whitespace in the resulting string
+            return json.dumps(obj, separators=(',', ':')).encode("utf-8")
```

Comment:
probably we should have such a function in utils.

Reply:
left as is